mac/start.sh

```diff
@@ -37,29 +50,59 @@ function start_datashare {
         -Dfile.encoding=UTF-8 \
         -Djava.system.class.loader=org.icij.datashare.DynamicClassLoader \
         -Djava.net.preferIPv4Stack=true \
+        ${DS_JAVA_OPTS} \
         -cp "./dist:/Applications/Datashare.app/Contents/Resources/datashare-dist-${datashare_version}-all.jar" org.icij.datashare.Main \
-        --dataDir "$datashare_data_path" \
-        --queueType MEMORY \
-        --busType MEMORY \
-        --dataSourceUrl jdbc:sqlite:file:"/Users/${USER}/Library/Datashare/dist/datashare.db" \
-        --settings ./dist/datashare.conf \
-        --mode EMBEDDED \
-        --browserOpenLink true \
-        --elasticsearchDataPath "$datashare_index_path" \
-        --pluginsDir "${datashare_plugins_path}" \
-        --extensionsDir "${datashare_extensions_path}"
+        "$@"
```

Comment:
This is very close to the docker entry point, nice !

Reply:
It's also very close to the Linux launcher!